src/ontodia/workspace/workspace.ts

```diff
@@ -264,6 +264,17 @@ export class Workspace extends Component<WorkspaceProps, State> {
         this.markup.paperArea.zoomToFit();
     }
 
+    clearAll = () => {
+        const elementIds = this.model.elements.map(e => e.id);
+        const batch = this.model.history.startBatch();
+
+        for (const elementId of elementIds) {
+            this.model.removeElement(elementId);
```

Comment:
This doesn't take into account maintaining `AuthoringState`, e.g. deleting new entity events. I recommend implementing it like this:
```ts
// clone elements to prevent modification while iterating
this.editor.removeItems([...this.model.elements]);
```